packages/android-client/FishjamClient/src/main/java/com/fishjamcloud/client/ReconnectionManager.kt

```diff
@@ -32,41 +32,37 @@ internal class ReconnectionManager(
 ) {
   private val listeners = mutableListOf<ReconnectionManagerListener>()
   private var reconnectAttempts = 0
-  private var reconnectionStatus = ReconnectionStatus.IDLE
+  private var reconnectionStatus = ReconnectionStatus.idle
 
   // attempts to reconnect if suitable
   fun onDisconnected() {
     if (reconnectAttempts >= reconnectConfig.maxAttempts) {
-      reconnectionStatus = ReconnectionStatus.ERROR
+      reconnectionStatus = ReconnectionStatus.error
       listeners.forEach { it.onReconnectionRetriesLimitReached() }
       return
     }
 
-    if (reconnectionStatus == ReconnectionStatus.WAITING) {
-      return
-    }
+    if (reconnectionStatus == ReconnectionStatus.reconnecting) return
```

Comment:
**question:** What do you think about always using brackets for if statements? It's easy to make an error accidentally and I think it doesn't really look that bad 😅

Reply:
I actually have no strong preferences for this. The if below didn't have brackets so I decided to make it consistent with less code. But I can add the brackets if only because Swift requires those brackets!